src/app/shared/components/template/services/template-variables.service.spec.ts

```diff
@@ -32,10 +33,7 @@ const MOCK_CONTEXT_BASE: IVariableContext = {
   calcContext: {
     globalConstants: {},
     globalFunctions: {},
-    thisCtxt: {
-      fields: MOCK_FIELDS,
-      local: {},
-    },
+    thisCtxt: {},
```

Comment:
Because calcContext gets overridden by `templateCalcService.getCalcContext()` as part of `evaluatePLHString()`,  the values here has no effect on the results of `evaluatePLHData()` in current test cases